data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,88 @@ mission "FW Epilogue: Danforth"
 			label end
 			`	After you talk for a while longer, Danforth wishes you luck and sends you on your way. "I'll be in touch if we ever need your assistance in the future," he says.`
 				decline
+
+
+
+mission "FW Epilogue: Barmy"
+	minor
+	landing
+	name `Weapon Testing`
+	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
+	source "Rust"
+	to offer
+		has "free worlds plot completed"
+	on offer
+		conversation
+			`As you complete your landing procedure you recall having come here to assist in some testing for new weapons, back in the thick of the war. Do you want to check in on Barmy Edwards, the lead engineer?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			scene "scene/loc0"
+			`	You give your name to the receptionist, and she directs you to the hangar where the weapons group does their design and prototyping. Barmy Edward shakes your hand warmly and says, "Ah, <last>. It's been too long! How fortunate for me, I'm actually just putting the finishing touches on a new prototype. You wouldn't be too busy to assist me in some practical testing would you?`
+			`	"I'm not sure about this, what would you have me do?" you say.`
+			`	"Not much out of the ordinary from our tests in the past," he says. "You'll find a ship of ours in orbit that goes by 'Hunk', it has been fitted with a few of these new weapons. Your job will be to fight it and disable it, then tell us what you think of its combat capabilities and transfer your ships logs so we can analyze the fight."`
+			choice
+				`	"Alright, I suppose for old times sake I can help out."`
+				`	"Sorry, I just wanted to check in and see how you were doing."`
+					goto catchup
+			`	He turns to you and says with a smirk, "Now mind you, we just want you to disable the ship, not destroy it. We've recommissioned it from retirement, so it's not top of the line but all the same I want to be able to keep using it for future tests. Understand?"`
+			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`
+				accept
+			label catchup
+			`	"I see, well I'm sure we can find someone else up to the task. Regardless captain <last>, I assure you this new weapon vastly improves on the outdated Plasma Cannon, this will put Kraz Cybernetics in the big leagues!"`
```

Comment:
```suggestion
			`	"I see. Well, I'm sure we can find someone else up to the task. Regardless Captain <last>, I assure you this new weapon vastly improves on the outdated Plasma Cannon. This will put Kraz Cybernetics in the big leagues!"`
```